motor_control/throttle/Button.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BUTTON_THROTTLE
+#define BUTTON_THROTTLE
+
+#include "base_throttle.h"
+
+class Button: public Throttle{
+    public:
+    Button(int ft);
```

Comment:
Nit: can you pick a better name for this constructor variable `ft`? I see below that it might mean time to full throttle, in which case it should probably be called `time_to_full_throttle`.